models/demos/t3000/mixtral8x7b/tests/test_mixtral_perf.py

```diff
@@ -59,8 +59,14 @@ def test_mixtral_model_perf(
 
     dtype = ttnn.bfloat8_b
 
+    batch_size = 32
+    # Although in decode-only mode change the max seqlen to 16k to avoid KV cache running out of memory size
+    max_seqlen = 16384
```

Comment:
should not max_seq_len be a function of the batch size?

Reply:
Yes. However this test is decode only, and for that one we are only running batch == 32.

For prefill test, we do batch_size in function of max seqlen.